recipes/opentelemac/conda_build_config.yaml

```diff
@@ -0,0 +1,32 @@
+zip_keys:
```

Comment:
Are you sure you need this? Conda-forge defaults are not working for you here?

Reply:
thanks for the review felipe.
We have problems with certain versions of python/numpy (details [there](https://github.com/nicogodet/telemac-mascaret-feedstock/issues/13)), hence the patch in `recipes/opentelemac/patches`. 

I assume @nicogodet tried the conda-forge defaults

Reply:
but I guess now we can remove this patch, since it has been integrated in main last year, correct?